src/backend/renderer/gles/mod.rs

```diff
@@ -2324,6 +2329,7 @@ impl Renderer for GlesRenderer {
 
         let current_projection = flip180 * transform.matrix() * renderer;
         let span = span!(parent: &self.span, Level::DEBUG, "renderer_gles2_frame", current_projection = ?current_projection, size = ?output_size, transform = ?transform).entered();
+        let vertices = std::mem::take(&mut self.vertices);
```

Comment:
I am not sure, why we have to move this, given the frame borrows the renderer anyway?

Reply:
Yeah, no reason to do this. I had something in mind, but can't exactly remember :shrug:

Reply:
done